src/tests/CredentialsTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+use AWS\CRT\Auth\AwsCredentials as AwsCredentials;
+use AWS\CRT\Auth\StaticCredentialsProvider as StaticCredentialsProvider;
+
+require_once('common.inc');
+
+final class CredentialsTest extends CrtTestCase {
+
+    public function testEmptyCredentials() {
+        $this->expectException(InvalidArgumentException::class);
+        $creds = new AwsCredentials(AwsCredentials::defaults());
+        $this->assertNotNull($creds, "Failed to create default/empty credentials");
+        $creds = null;
+    }
+
+    private function credentialsOptions() {
```

Comment:
standard would be to either use a setup/teardown function (on larger scoped variables) or change the name of this function to `getCredentialsConfig()`